grammar.js

```diff
@@ -905,6 +925,99 @@ module.exports = grammar({
       ')',
     ),
 
+    window_partition_by_expression: $ => seq(
+        $.keyword_partition_by,
+        $.identifier,
+    ),  
+
+    window_order_by_expression: $ => seq(
```

Comment:
`using` and `nulls first/last` are allowed in statement `order by` clauses as well, so it'd be good to update that and reuse it instead of creating a near clone!